xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/XMLUtils.java

```diff
@@ -276,15 +277,15 @@ public static String escapeAttributeValue(Object content)
     }
 
     /**
-     *  Escapes some XML special characters (&amp; and &lt;) in a <code>String</code> using numerical XML
-     *  entities, so that the resulting string can safely be used in HTML pages.
+     * Escapes XML special characters in a {@code String} using numerical XML entities, so that the resulting string
+     * can safely be used as an XML text value. Specifically, escapes &lt;, and &amp;.
```

Comment:
"as an XML text value." --> "as an XML element text value (e.g. {@code Jim & John} will be escaped and can thus be put inside an XML tag, such as the {@code p} tag, as in {@code <p>Jim &amp; John</p>}).".

Should use `{@code ...}` constructs for `&lt;` and `&amp;`.

Reply:
Javadoc updated here https://github.com/xwiki/xwiki-commons/pull/102/files#diff-c1eed56adbad8fd67a8e196907ce032fR287